nvflare/private/fed/app/deployer/simulator_deployer.py

```diff
@@ -15,14 +15,13 @@
 import shutil
 import tempfile
 
-from nvflare.apis.event_type import EventType
+from nvflare.fuel.f3.cellnet.cell import Cell
+from nvflare.fuel.f3.mpm import MainProcessMonitor as mpm
 from nvflare.fuel.utils.network_utils import get_open_ports
-from nvflare.private.fed.app.utils import create_admin_server
-from nvflare.private.fed.client.admin import FedAdminAgent
-from nvflare.private.fed.client.admin_msg_sender import AdminMessageSender
-from nvflare.private.fed.client.client_req_processors import ClientRequestProcessors
-from nvflare.private.fed.client.fed_client import FederatedClient
-from nvflare.private.fed.simulator.simulator_client_engine import SimulatorClientEngine, SimulatorParentClientEngine
+from nvflare.private.fed.app.server.server_train import create_admin_server
```

Comment:
We should have `create_admin_server` inside the `nvflare.private.fed.app.utils` as before.
And both `server_train` or `simulator_deployer` should import from there

Reply:
This will be addressed when I create the PR for simulator.